lib/utils/parse_link_header.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(link) {
+  var links = {};
+  if (!link) {
+    return links;
+  }
+  var parts = link.split(",");
+  // Parse each part into a named link
+  parts.forEach(function(part) {
+    var section = part.split(';');
+    if (section.length < 2) {
+      return;
+    }
+    var url = section[0].match(/<(.*)>/)[1];
+    var name = section[1].match(/rel="?([^"]*)"?/)[1];
+    links[name] = url;
+  });
+  return links;
+};
```

Comment:
Do we need to do any decoding on the header value?

Reply:
@mkdynamic this should be the `Link` header string coming from the response, which decoding  are you thinking about?

Reply:
Just wondering if the header value that is sent in as the argument is already decoded or not. Like, if the header value contained non-ASCII characters they get encoded when being added to the response (I believe). I suspect any decoding gets handled by the library we are using to read response headers.

Reply:
Yeah, just did a test, say our API server sends a non-ASCII char in the link header, this client will properly retrieve it and pass it back to the server.

Reply:
Great, thanks for double checking that!